app/views/documents/_new.html.erb

```diff
@@ -2,6 +2,8 @@
     <%= f.label :name %>
     <%= f.text_field :name,:required => 'required' %>
     <%= f.label :attachment %>
+    <td> (.pdf .doc .docx .csv .jpg .xlsx .xls .xlt .txt .png .heic
```

Comment:
I think this looks a bit out of place just written in text format above the file selector box.

You can do something like this https://apidock.com/rails/ActionView/Helpers/FormHelper/file_field
Which tells the file upload box what to allow. Look at the `accepts: ... ` option. Otherwise, I think you could just have the error message telling you which files you can upload.

Reply:
I removed it, and added the "accept"!